templates/webapi/step/edit.html.ep

```diff
@@ -152,6 +152,14 @@
                                     <input tupe="input" name="workaround_desc" id="input_workaround_desc" class="form-control" value="">
                                 </span>
                             </p>
+                            % if ($git_enabled) {
+                            <p>
+                                <label for="needleeditor_commit_message">Commit message:</label>
```

Comment:
We should make it clear to the user that the git commit message details are optional. How about

s/Commit message:/Optional commit message details (leave empty for default text):/

Reply:
If it is left empty it looks like this:

![screenshot_20210223_115520](https://user-images.githubusercontent.com/10248953/108834084-0440cb00-75ce-11eb-9eaa-60c76f6c539f.png)

I've mentioned the use of the placeholder text before. I think with this we can omit the verbose explanation (which wouldn't even tell the user what the default text is).

Reply:
Sure, I like the placeholder text but for me as a user it's not obvious if the grey text will be used by default or if it's just an example shown to the user but never ending in the git commit message. Just compare that to the text field in the lower right with the text "Add new tags here". This text is not automatically added as a tag if the user does not overwrite it, it's an explanation that is just shown to the UI user, not ending in the needle JSON.